vault/logical_system_activity_write_testonly.go

```diff
@@ -36,7 +36,7 @@ func (b *SystemBackend) activityWritePath() *framework.Path {
 			},
 		},
 		Operations: map[logical.Operation]framework.OperationHandler{
-			logical.CreateOperation: &framework.PathOperation{
+			logical.UpdateOperation: &framework.PathOperation{
```

Comment:
I hid whitespace and this is the only change that remained. Is this intended?

Reply:
I almost snuck out of this undetected! @cipherboy has a new/better PR here: https://github.com/hashicorp/vault/pull/21682 to address the issue more robustly (and less erroneously) 😄.